Expensify/App
# [User Settings] Add new Profile view

### Details
- Added new personal details update form to profile page
- Created new checkbox component

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify.cash/issues/1711

### Tests
- Go to settings/profile, modify details
- Click "save" and refresh app to confirm changes took effect

### Tested On


### Video



https://user-images.githubusercontent.com/12075600/111050743-49089680-8403-11eb-8001-888d5b39aed8.mov



#### Web

![Screen Shot 2021-03-13 at 1 31 30 PM](https://user-images.githubusercontent.com/12075600/111049516-69842100-8402-11eb-9f4e-b330e41a9fc5.png)

#### Mobile Web

<img width="494" alt="Screen Shot 2021-03-13 at 1 30 03 PM" src="https://user-images.githubusercontent.com/12075600/111048526-2924a300-8402-11eb-99df-4d6fc88835ae.png">

#### Desktop

<img width="1312" alt="Screen Shot 2021-03-13 at 1 32 17 PM" src="https://user-images.githubusercontent.com/12075600/111048628-2fb31a80-8402-11eb-8041-ebcbc9623745.png">

#### iOS

<img width="494" alt="Screen Shot 2021-03-13 at 1 31 13 PM" src="https://user-images.githubusercontent.com/12075600/111048772-393c8280-8402-11eb-8633-f52fe1bcef47.png">

#### Android

<img width="464" alt="Screen Shot 2021-03-13 at 1 32 43 PM" src="https://user-images.githubusercontent.com/12075600/111048929-42c5ea80-8402-11eb-8a1f-b711e9e638b1.png">
